src/styles/pages/cp/_requests.scss

```diff
@@ -54,8 +54,11 @@ $page: "p-cp-requests";
 			}
 		}
 
-		td:nth-last-child(1) {
-			padding-left: 0;
+		td, th {
+			&:nth-last-child(1) {
+				display: none; // Hidden, see ARC-409
```

Comment:
Misschien even aanhalen dat dit over de extra acties gaat. Is niet meteen duidelijk aangezien je ze nu niet meer ziet 🙈